src/NServiceBus.Callbacks/SynchronousLocalOptions.cs

```diff
@@ -0,0 +1,49 @@
+namespace NServiceBus
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    ///     Allows the users to control how the send is performed
+    /// </summary>
+    public class SynchronousLocalOptions
```

Comment:
Think I've manage to sway @SimonCropp and @danielmarbach to skip the local option all together but I'll raise a separate pull for that later